### DISCUSSION TITLE
工作区笔记无法编辑（两台电脑同步）

### DISCUSSION THREAD

**Discussion by iamlgdm** [windingwind/zotero-better-notes#896] [Q&A]
如题，一台电脑写完保存以后，到另一个地方换了电脑写就发现无法编辑

**Comment by windingwind** [comment]
检查笔记是否在回收站

**Reply by iamlgdm** [reply]
![image](https://github.com/windingwind/zotero-better-notes/assets/158118731/04f03846-5f43-43de-ac33-f3167f456246)
不在回收站。。。还是编辑不了
